src/libraries/System.Private.CoreLib/src/System/DateTime.cs

```diff
@@ -697,7 +697,7 @@ public bool Equals(DateTime value)
         //
         public static bool Equals(DateTime t1, DateTime t2)
         {
-            return t1.Ticks == t2.Ticks;
+            return t1 == t2;
```

Comment:
Looks like we have similar pattern in TimeOnly class: https://github.com/dotnet/runtime/blob/c9e2675c9e2c15b483997562ba854d72b19b2034/src/libraries/System.Private.CoreLib/src/System/TimeOnly.cs#L317

Reply:
TimeOnly's one compare two fields so it's ok. DateTime compared two computed properties